src/components/landing/CenteredHero/HeroForm.js

```diff
@@ -0,0 +1,191 @@
+      when we support yours.
+    </p>
+  </>
+);
+
+const HeroForm = () => {
+  const [honeypotText, setHoneypotText] = useState('');
+  const [submitting, setSubmitting] = useState(false);
+  const [email, setEmail] = useState('');
+  const [scmTool, setScmTool] = useState('github-cloud');
+  const [modalOpen, setModalOpen] = useState(false);
+  const [showError, setShowError] = useState(false);
+  const netlifyFormName = FORM_NAMES.tryRoadieHomepage;
+  const buttonText = 'Try Roadie Backstage';
+
+  let disabled = submitting;
+
+  const onSubmit = async (e) => {
+    e.preventDefault();
+
+    if (disabled) return false;
+    setSubmitting(true);
+    setShowError(false);
+
+    const resp = await submitToNetlifyForms({
+      email,
+      scmTool,
+      netlifyFormName,
+      honeypotText,
+      subToNewsletter: false,
+      agreeToPolicies: true,
+      submitButtonLabel: buttonText,
+      recaptchaResponse: '',
+    });
+
+    setSubmitting(false);
+
+    if (resp.ok) {
+      setModalOpen(true);
+      setEmail('');
+    } else {
+      setShowError(true);
+      console.log('error', resp);
+    }
+
+    setSubmitting(false);
+  };
+
+  return (
+    <>
+      <SubmissionSuccessModal
+        scmTool={scmTool}
+        handleCloseModal={() => setModalOpen(false)}
+        modalOpen={modalOpen}
+        positiveTitle="We'll be in touch!"
+        positiveBody={<SubmissionSuccessPositiveBody />}
+        negativeTitle="Watch this space!"
+        negativeBody={<SubmissionSuccessNegativeBody />}
+      />
+      <Form
+        onSubmit={onSubmit}
+        name={netlifyFormName}
+        honeypotValue={honeypotText}
+        onHoneypotChange={setHoneypotText}
+        buttonText={buttonText}
+        disableRecaptcha={true}
+        className="grid grid-cols-1 gap-y-6 sm:grid-cols-6 sm:gap-x-8"
+      >
+        <TextField
+          id="get-instance-email-input"
+          label="Work email"
+          name="email"
+          type="email"
+          autoComplete="email"
+          fullWidth={true}
+          onChange={setEmail}
+          value={email}
+          required={true}
+          color="secondary"
+        />
+        <div className="sm:col-span-2 mt-4">
+          <ScmToolSelect
+            label="Primary source code host"
+            onChange={setScmTool}
+            currentValue={scmTool}
+            idPrefix="get-instance-"
+            color="secondary"
+          />
+        </div>
+        <div className="sm:col-span-2 flex items-end">
+          <Button
+            type="submit"
+            color="secondary"
+            size="medium"
+            fullWidth={true}
+            text={buttonText}
+            disabled={disabled}
+            required={true}
+            className="font-bold tracking-wide"
+          />
+        </div>
+
+        <div className="sm:col-span-4">
+          <p className="text-sm text-white opacity-60">
```

Comment:
If this legal stuff is duplicated on the other form, it would be worth extracting it into a component.

Reply:
It's not duplicated.